nixos/modules/services/x11/desktop-managers/gnome3.nix

```diff
@@ -45,10 +47,19 @@ let
 
   flashbackEnabled = cfg.flashback.enableMetacity || length cfg.flashback.customSessions > 0;
 
-in {
+in
+
+{
 
   options = {
 
+    services.gnome3 = {
+      core-os-services.enable = mkEnableOption "essential services for GNOME3";
+      core-shell.enable = mkEnableOption "GNOME Shell services";
+      core-utilities.enable = mkEnableOption "GNOME core utilities";
+      games.enable = mkEnableOption "GNOME games";
```

Comment:
I wanted to document what packages get enabled with `relatedPackages` but I need some sort of hack like
* https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/x11/xserver.nix#L253

It would also be nice in the case of `core-utilities` if there's a convenient way to show what `programs` options get enabled.